install_config/configuring_authentication.adoc

```diff
@@ -65,9 +65,68 @@ Basic authentication challenges are only sent if a `X-CSRF-Token` header is
 present on the request. Clients that expect to receive Basic `WWW-Authenticate`
 challenges should set this header to a non-empty value.
 
-.^|`login`     | When *true*, unauthenticated token requests from web clients
+|`login`     | When *true*, unauthenticated token requests from web clients
 (like the web console) are redirected to a login page backed by this provider.
 Not supported by all identity providers.
+
+|`mappingMethod`  | Defines how new identities are mapped to users when they login. See link:#mappingmethod-parameter[below] for more information.
+|===
+
+[NOTE]
+When adding or changing identity providers, you can map identities from the new
+provider to existing users by setting the `*mappingMethod*` parameter to
+`*add*`.
+
+[[mappingmethod-parameter]]
+
+=== The `*mappingMethod*` Parameter
```

Comment:
Suggestions for section title: "Mapping Identities" or "Mapping Identities to Users". And if changed, update anchor and xrefs accordingly.